ClientApp/viking-vault/src/components/SideBar.tsx

```diff
@@ -34,25 +34,28 @@ class SideBar extends React.Component<any, ISideBarState> {
         return(
             <div className={"sidebar-" + this.state.show} id="mySidebar">
                 <div className="regular-div">
-                    <p className="p"> <img className="img" src={dashboard} alt=""/> My Dashboard  <img src={arrow} className = {this.state.show ? "transform-none img" : "transform-reverse img"} alt="" onClick={this.clickHandler}/></p>
```

Comment:
Could you maybe give more suggestive names for the classes of `regular-div` and `p`?, for example you could name it `standard-container` if that's what it does..?

Reply:
It's okay, I erased all those p's anyway :3